x-pack/plugins/security_solution/public/common/components/exceptions/edit_exception_modal/index.tsx

```diff
@@ -204,7 +206,11 @@ export const EditExceptionModal = memo(function EditExceptionModal({
           </ModalHeaderSubtitle>
         </ModalHeader>
 
-        {!isSignalIndexLoading && (
+        {(addExceptionIsLoading || indexPatternLoading || isSignalIndexLoading) && (
+          <Loader data-test-subj="loadingAddExceptionModal" size="xl" />
```

Comment:
Could we use a different `data-test-subj` for this element?

Reply:
fixed